### DISCUSSION TITLE
Silly idea I have: multiplayer?

### DISCUSSION THREAD

**Discussion by likeproblem** [likeproblem/RestockRecolour#14] [Ideas]
@Spartwo @UncleMateo @EmberSkyMedia just though on one thing, do you wanna guys meet in multiplayer(specifically LMP) on weekends? You know, almost month aniversary since we started and 30% parts done! And we get to play with recolour parts :)

(I just thought it would be fun, server on me just in case)

**Comment by likeproblem** [comment]
![2xoq9krxvir91](https://github.com/user-attachments/assets/4668cf27-ee95-42d5-a088-0c417e2f07ff)

**Comment by EmberSkyMedia** [comment]
Maybe next weekend, this weekend is packed for me.

**Reply by UncleMateo** [reply]
Okay so I'm gonna be 100% honest.  Since I started The Felota Project ~2-3 months ago, I haven't really been _playing_ KSP.  In fact, for a month or two before that, I was playing less and less and working on a planet pack (which ultimately was shelved).  Basically, over time, a greater and greater percentage of my KSP-time started being consumed by modding, and less and less was actually spent playing.  So when you asked this, I had a hard time figuring out how to respond.
 
Maybe I need to shake things up a little, though.  Maybe I'm running out of steam at this pace, and I just need to dial back on the throttle a bit, let that overheating die down, ya know?  To use kerbal metaphors.

**Reply by EmberSkyMedia** [reply]
I started a YT gaming channel so I gave myself a reason to play (make content) or I’d likely never play LOL… always modding something somewhere.

**Comment by EmberSkyMedia** [comment]
Next Saturday is ExtraLife where I raise money for Hospital Charity, so I’ll be streaming that day… likely SFS as on my channel that drives quite a bit of traffic when I stream it (vs. Kerbal).

**Reply by likeproblem** [reply]
Since you are busy on Saturday, maybe we should play on Sunday then?

**Reply by EmberSkyMedia** [reply]
That’s possible

**Comment by likeproblem** [comment]
@EmberSkyMedia @UncleMateo are we set for multiplayer this Sunday?
